pkg/kademlia/kademlia_test.go

```diff
@@ -35,6 +34,12 @@ const (
 	defaultAlpha = 5
 )
 
+type testNode struct {
```

Comment:
why is this named testNode?

Reply:
because it is returned by the testNode function

Reply:
I thought it would be better to make a struct to hold the variables rather than return 4 variables out of testNode

Reply:
gotcha